core/chains/evm/gas/rollups/op_l1_oracle_test.go

```diff
@@ -101,7 +102,8 @@ func TestOPL1Oracle_ReadV1GasPrice(t *testing.T) {
 				assert.Nil(t, blockNumber)
 			}).Return(common.BigToHash(l1BaseFee).Bytes(), nil).Once()
 
-			oracle, err := newOpStackL1GasOracle(logger.Test(t), ethClient, chaintype.ChainOptimismBedrock, oracleAddress)
+			daOracle := CreateTestDAOracle(t, toml.OPStack, "0x0000000000000000000000000000000000001234", "")
```

Comment:
Could we also update the random address here and below with `utils.RandomAddress().String()` also?
```suggestion
			daOracle := CreateTestDAOracle(t, toml.OPStack, utils.RandomAddress().String(), "")
```

Reply:
@amit-momin updated 👍